app/src/util/tools.ts

```diff
@@ -501,13 +501,16 @@ export const limitDecimalPlaces = (value: string, decimals: number) => {
 }
 
 export const formatNumber = (number: string, decimals = 2): string => {
-  if (number.length < 1) {
-    return `0${decimals > 0 ? '.' + '0'.repeat(decimals) : ''}`
-  }
-
   const fixedInt = parseFloat(number.split(',').join('')).toFixed(decimals)
   const splitFixedInt = fixedInt.split('.')[0]
   const formattedSubstring = splitFixedInt.replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,')
+  if (number.length < 1) {
+    return `0${decimals > 0 ? '.' + '0'.repeat(decimals) : '0'}`
```

Comment:
Not sure if there's a reason for setting `: '0'` instead of `: ''` here, but if I do `formatNumber('', 0)` (the only situation this should fire), then it returns '00' which is probably not what we want

Reply:
thanks